chapter12/pxd/ex1.pl

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use feature "say";
+
+
+if(scalar @ARGV == 0) {
+	die "You need to provide at least one file as argument";
+}
+
+my @file_list = @ARGV;
+
+sub check_file_permissions {
+	my $file = shift;
+
+	my %permissions = (
```

Comment:
Cool, I had wanted to put the operations in to a hash, didn't think of putting the results in one